modules/samples/connectedcup/component/agent/src/main/webapp/index.jsp

```diff
@@ -1,140 +1,134 @@
-                var token = '<%=request.getSession().getAttribute("token")%>';
-                var url = "https://localhost:9443/connectedcup/controller/ordercoffee?deviceId=" + deviceId +"&deviceOwner=" +
-                        deviceOwner;
-
-                $.ajax({
-                    type: 'POST',
-                    url: url,
-                    headers: {
-                        "Authorization" : "Bearer " + token
-
-                    }
-                });
-
-            });
-
-            function sendData()
-            {
-                var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
-                var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
-                var tempPayload = "temperature:" + $( "#temperature_level" ).slider( "value" );
-                var levelPayload = "coffeelevel:" + $( "#slider-vertical" ).slider( "value" );
-                $.post( "/connected-cup-agent/push_temperature?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
-                        "&payload=" + tempPayload);
-                $.post( "/connected-cup-agent/push_level?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
-                        "&payload=" + levelPayload);
-                setTimeout(sendData, 5000);
-            }
-
-            sendData();
-
-        </script>
-    </body>
-</html>
+</div>
+
+<script src="js/libs/jquery.min.js"></script>
+<script src="js/coffee.js"></script>
+<script src="js/libs/htmlpreview.min.js"></script>
+<script>HTMLPreview.replaceAssets();</script>
+<script>
+    $("#order-cup").click(function() {
+
+        var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
+        var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
+        var token = '<%=request.getSession().getAttribute("token")%>';
+        var url = "/connectedcup/controller/ordercoffee?deviceId=" + deviceId +"&deviceOwner=" +
+                  deviceOwner;
+
+        $.ajax({
+                   type: 'POST',
+                   url: url,
+                   headers: {
+                       "Authorization" : "Bearer " + token
+
+                   }
+               });
+
+    });
+
+    function sendData()
+    {
+        var deviceId = '<%=request.getSession().getAttribute("deviceId")%>';
+        var deviceOwner = '<%=request.getSession().getAttribute("deviceOwner")%>';
+        var tempPayload = "temperature:" + temperature;
+        var levelPayload = "coffeelevel:" + coffee_amount;
+        $.post( "/connected-cup-agent/push_temperature?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
+                "&payload=" + tempPayload);
+        $.post( "/connected-cup-agent/push_level?deviceId=" + deviceId +"&deviceOwner=" + deviceOwner +
+                "&payload=" + levelPayload);
+        setTimeout(sendData, 5000);
+    }
+
+    sendData();
+
+</script>
+</body>
+</html>
```

Comment:
Yes we are using. This is device agent web application. Not a part of devicemgt app